<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgEdgeEdit</class>
 <widget class="QDialog" name="CQDlgEdgeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Operations on selected edges</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqMakePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Extract path</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Edge mode properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqShowHiddenEdges">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show hidden edges</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAutoFollowing">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Automatic edge following</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>45</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. edge angle [deg]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqEdgeAngle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>45</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>65</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. dir. change angle [deg]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqDirectionChangeAngle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>65</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Edge selection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqClearSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Clear selection</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqInvertSelection">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Invert selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>401</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqShowHiddenEdges</tabstop>
  <tabstop>qqAutoFollowing</tabstop>
  <tabstop>qqEdgeAngle</tabstop>
  <tabstop>qqDirectionChangeAngle</tabstop>
  <tabstop>qqClearSelection</tabstop>
  <tabstop>qqInvertSelection</tabstop>
  <tabstop>qqMakePath</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
